packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -194,11 +194,14 @@ export default class TableResize implements EditorPlugin {
                                 }
                                 if (verticalInserterTd) {
                                     this.setCurrentTd(null);
-                                    this.setCurrentInsertTd(
-                                        ResizeState.Vertical,
-                                        verticalInserterTd,
-                                        map.rect
-                                    );
+                                    // we hide the inserter if left mouse button is pressed
+                                    if (e.buttons == 0) {
```

Comment:
Actually, why not just do this check at very beginning of this function (line 130), and return directly if buttons is not 0? Not need to do all further calculation.

Reply:
Oh I see, so in the dragging scenario, onmousemove still need to work.